yt_dlp/extractor/urplay.py

```diff
@@ -29,19 +30,49 @@ class URPlayIE(InfoExtractor):
             'episode': 'Om vetenskap, kritiskt tänkande och motstånd',
             'age_limit': 15,
         },
+    }, {
+        'url': 'https://urplay.se/program/222967-en-foralders-dagbok-mitt-barn-skadar-sig-sjalv',
+        'info_dict': {
+            'id': '222967',
+            'ext': 'mp4',
+            'title': 'En förälders dagbok : Mitt barn skadar sig själv',
+            'description': 'md5:9f771eef03a732a213b367b52fe826ca',
+            'thumbnail': r're:^https?://.+\.jpg',
+            'timestamp': 1629676800,
+            'upload_date': '20210823',
+            'series': 'En förälders dagbok',
+            'duration': 1740,
+            'age_limit': 15,
+            'episode_number': 3,
+            'categories': ['Hälsa & relationer / Familj & sociala relationer',
+                           'Hälsa & relationer / Psykisk ohälsa'],
+            'tags': ["Familjesociologi",
+                     "Föräldrar och barn",
+                     "Föräldraskap",
+                     "Psykiatri",
+                     "Samhällsvetenskap",
+                     "Självdestruktivt beteende",
+                     "Sociologi"],
```

Comment:
```suggestion
            'categories': 'count:2',
            'tags': 'count:7',
```

Reply:
Good catch. Actually, I was under the impression that UR Play reorganized their categories recently, so even with a count, this may break. But your suggestion is safer.